src/css/profile/mobile/changeable/common/coverflow.less

```diff
@@ -1,5 +1,10 @@
+// CoverFlow style
 .ui-coverflow {
-	&:focus {
-		background-color: @color_focus_background;
-	}
-}
+	background-image: url("images/interactive/coverflow.png");
```

Comment:
Widget looks little strange.
![image](https://user-images.githubusercontent.com/29534410/63344821-9ae55980-c351-11e9-944a-bd8a067161b5.png)

Reply:
I tested at WAPPL-VSC.
It worked fine.
How did you test the images you attached?

Reply:
Also, CoverFlow is only used for mobile profiles.

Reply:
@DohyungLim 
CoverFlow  is common profile and need to work in the wearable.
IAnd it also have to working the Tizen SDK Emulator.